src/ai/api/openaiApi.ts

```
@@ -0,0 +1,142 @@
+}
+
+/**
+ * Extract token usage information from an OpenAI API response.
+ * @param response - The OpenAI API response object.
+ * @returns Token usage information or null if unavailable.
+ */
+export function extractOpenAITokenUsage(response: OpenAIResponse): TokenUsage | null {
+  if (response && response.usage) {
+    return {
+      inputTokens: response.usage.prompt_tokens || 0,
+      outputTokens: response.usage.completion_tokens || 0,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)